Formula/glibc.rb

```diff
@@ -47,7 +47,7 @@ class MakeRequirement < GlibcBaseRequirement
   fatal true
   satisfy(build_env: false) { which(TOOL).present? }
   TOOL = "make".freeze
-  VERSION = "3.79 (or later)".freeze
+  VERSION = nil
```

Comment:
Can `nil` be made implicit here instead?

Reply:
That sounds fine to me but I'm not sure how to implement that.  Does it need to be handled in `GlibcBaseRequirement`?  Right now it just takes `self.class::VERSION` - I'm not sure if will implicitly use `nil` if VERSION is undefined by the derived class instance.

Reply:
Yeh, it make make more sense to use `def version` instead. Fine for this to be punted on though, not a big deal.

Reply:
Yeah maybe it's best to punt here for now.  I'm totally willing to try to fix this up later.